src/lib/content.js

```diff
@@ -123,7 +123,13 @@ function loadProfiles(profileSet, list, csrf, hidesHistory, hidesAccountId) {
     if (!hidesAccountId) name += '  |  ' + item.aws_account_id;
 
     var color = item.color || 'aaaaaa';
-    var actionHost = window.location.host.endsWith('.amazonaws-us-gov.com') ? 'signin.amazonaws-us-gov.com' : 'signin.aws.amazon.com';
+    if (window.location.host.endsWith('.amazonaws-us-gov.com')) {
+        var actionHost = 'signin.amazonaws-us-gov.com';
+    } else if (window.location.host.endsWith('.amazonaws.cn')) {
+        var actionHost = 'signin.amazonaws.cn';
+    } else {
+        var actionHost = 'signin.aws.amazon.com';
+    }
```

Comment:
Please change the following
```suggestion
    let actionHost;
    if (window.location.host.endsWith('.amazonaws-us-gov.com')) {
        actionHost = 'signin.amazonaws-us-gov.com';
    } else if (window.location.host.endsWith('.amazonaws.cn')) {
        actionHost = 'signin.amazonaws.cn';
    } else {
        actionHost = 'signin.aws.amazon.com';
    }
```